src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String operation;
+
+    Operation(String operation) {
+        this.operation = operation;
+    }
+
+    public static Operation getOperation(String operation) {
+        switch (operation) {
```

Comment:
each of your enums have field named `operation`
you can go through all enum values with Stream API and search by this field